### ISSUE TITLE
Open API Security Headers not respected in documentation and collection generation

### ISSUE THREAD

**Issue by mahirk** [a85/POSTMan-Chrome-Extension#9004]
**Describe the bug**
Postman API to Collection generation does not support the global level security header, or the API specific security level headers as Auth strategies. The strategies are described here: https://swagger.io/docs/specification/authentication/ 

**To Reproduce**
1. Go to "Browse"
2. Click on "Create new API" with "Open API 3.0"
3. Enter test names and open "Define" 
4. A sample schema should be present. Replace the schema with
```
openapi: 3.0.0
info:
  version: '1'
  title: Test API
servers:
  - url: 'http://localhost:3000'
paths:
  /user:
    get:
      summary: 'Sample endpoint: Returns details about a particular user'
      operationId: listUser
      tags:
        - user
      parameters:
        - name: id
          in: query
          description: ID of the user
          required: true
          schema:
            type: integer
            format: int32
      responses:
        '200':
          description: 'Sample response: Details about a user by ID'
          headers:
            x-next:
              description: A link to the next page of responses
              schema:
                type: string
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/User'
        default:
          description: Unexpected error
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Error'
components:
  schemas:
    User:
      type: object
      required:
        - id
        - name
      properties:
        id:
          type: integer
          format: int64
        name:
          type: string
        tag:
          type: string
    Error:
      type: object
      required:
        - code
        - message
      properties:
        code:
          type: integer
          format: int32
        message:
          type: string
  securitySchemes:
    apiKey:
      type: apiKey
      in: header
      name: X-API-KEY
    appId:
      type: apiKey
      in: header
      name: X-APP-ID
security:
  - apiKey: []
    appId:  []   
```
5. Save
6. Click on "Develop"
7. Create Documentation
8. Open documentation

**Expected behavior**
With a global Authentication strategy like above, the documentation should present the API as authenticated and show the strategy options in the "Authentication" section. This also applies to the created API collection. 

**Screenshots**
Current:
![Screen Shot 2020-08-31 at 4 17 33 PM](https://user-images.githubusercontent.com/5751414/91777948-0b01fe80-eba6-11ea-8a9f-6c8a375d5bc7.png)

Expected:
![Screen Shot 2020-08-31 at 4 21 02 PM](https://user-images.githubusercontent.com/5751414/91777949-0ccbc200-eba6-11ea-8167-658b93e75151.png)

**App information (please complete the following information):**
 - App Type: Chrome App
 - Postman Version: Latest on browser
 - OS: [e.g. macOS Mojave 10.14.6]

**Comment by VShingala** [comment]
@mahirk Thanks for reporting your issue, We were already in process of fixing issue. While converting from OpenAPI definition to collectionm current behaviour was to add header/query mentioned in `apiKey` security scheme to all requests. And PR mentioned above will add support to include it as Request Auth.

**Comment by mahirk** [comment]
Thank you @VShingala. I noticed the prior issue closed but missed the PR, thank you for referencing them!

**Comment by mahirk** [comment]
@VShingala I see the PR is merged. I'll close this issue and wait for the release. Thank you!

**Comment by mahirk** [comment]
Hey @VShingala I apologize I never asked: When would this be released?

**Comment by VShingala** [comment]
@mahirk This will be released with next release, which will be available on start of next week.

**Comment by VShingala** [comment]
@mahirk This issue should be fixed with the latest App release v7.33. Closing this issue for now, feel free to reopen this issue for any further problems
